### ISSUE TITLE
Our various application download filenames are vague and confusing

### ISSUE THREAD

**Issue by benjiwheeler** [LLK/scratch-www#3603]
When you try to download various applications or files from the Scratch website -- Scratch Link, Scratch Desktop, Microbit hexfile, etc., many of them have cryptic names like "mac".

E.g., https://downloads.scratch.mit.edu/link/mac.zip on the page https://scratch.mit.edu/microbit .

I think this is confusing -- even with all my familiarity with our site, it gives me pause when the suggested file for what I'm downloading is that out of line with my expectations. 

We should consider renaming these with more informative names, such as "Scratch Link for Mac".

**Comment by Accio1** [comment]
I agree that the files should have a more specific name. If someone has lots of files on their computer, they might lose the file after downloading it, because they would be looking for a file which contains the word "scratch", not just "windows.zip" or "mac.zip".

**Comment by kchadha** [comment]
This is no longer an issue for the Scratch desktop applications for Windows and Mac.

It is still an issue for Scratch Link and the Microbit hexfile (tracking this one in #3602)
